core/cache/model.go

```diff
@@ -62,7 +65,19 @@ func (m *Model) Application(appName string) (*Application, error) {
 		return nil, errors.NotFoundf("application %q", appName)
 	}
 	return app, nil
+}
 
+// Unit returns the unit for the input identifier.
+// If the unit is not found, a NotFoundError is returned.
+func (m *Model) Unit(unitId string) (*Unit, error) {
```

Comment:
For consistency, isn't unitId really unitName?  If not, what's the difference between the identifier and a unit name?

Reply:
Fair point; I've renamed it.